contrib/test/integration/system.yml

```diff
@@ -59,6 +58,11 @@
   async: '{{ 20 * 60 }}'
   poll: 10
 
+- name: install container-selinux
+  shell: |
+    curl -o container-selinux.rpm https://kojipkgs.fedoraproject.org//packages/container-selinux/2.109.0/2.gitdb771da.fc30/noarch/container-selinux-2.109.0-2.gitdb771da.fc30.noarch.rpm
+    yum install container-selinux.rpm -y
+
```

Comment:
Using yum to install a Fedora package seems a dangerous given the playbook runs against multiple platforms.  Shouldn't this task be limited or specified better using a `when: ansible_distribution in Foo` type attribute?

Reply:
Thanks for having a look :pray: This change can be more is less reverted since it does not solve anyhow the problem. The main issue seems that runc rc-8 seems incompatible with the older RHEL/Fedora versions (see [these logs](https://storage.googleapis.com/origin-federated-results/pr-logs/pull/cri-o_cri-o/2613/test_pull_request_crio_ami_fedora/472/artifacts/testout.txt)) Do you think that it would be possible to update the base images to some recent fedora and rhel versions?